index.js

```diff
@@ -42,13 +37,28 @@ ourBooks.forEach((book) => {
 
 // Remove Books
 function RemoveBook(id) {
-  ourBooks = ourBooks.filter(book => {
+  ourBooks = ourBooks.filter((book) => {
     if (id === book.id) {
       return false;
     } else {
       return true;
     }
-  })
-  console.log(ourBooks);
+  });
+  localStorage.setItem("save", JSON.stringify(ourBooks));
+  location.reload();
 }
-// Add Books
\ No newline at end of file
+// Add Books
+
+const AddBtn = document.querySelector("#add_Button");
+AddBtn.addEventListener("click", () => {
+  let name = document.querySelector("#title").value;
+  let author = document.querySelector("#author").value;
+  let id = ourBooks[ourBooks.length - 1].id + 1;
+  ourBooks.push({
+    id,
+    name,
+    author,
+  });
+  localStorage.setItem("save", JSON.stringify(ourBooks));
+  location.reload();
+});
```

Comment:
- [ ] Great job so far 👍 books can be added, removed and the data is being stored in the local storage of the browser 💯 but when I remove all the books and try adding a book again, I get this error on the console and the book is not added. Kindly address this issue 😄 
- Console error:
![Annotation 2022-04-18 121512](https://user-images.githubusercontent.com/85113567/163801091-d1c6241e-d507-4277-9bf5-24ac822ebe27.jpg)